### DISCUSSION TITLE
The window download page is not working.

### DISCUSSION THREAD

**Discussion by JinWallner** [MiKTeX/miktex#1320] [General]
Hi,

The window download link is not working. Can someone fix it?

**Comment by EPRLManager** [comment]
same, i had to get old version from the link that pops up, https://ctan.math.washington.edu/tex-archive/

**Comment by InvisibleMass** [comment]
same here in Hungary, where e.g.
https://miktex.org/download/ctan/systems/win32/miktex/setup/windows-x64/basic-miktex-23.4-x64.exe
is redirected to
https://mirror.szerverem.hu/ctan/systems/win32/miktex/setup/windows-x64/basic-miktex-23.4-x64.exe
which reports a 404
